src/components/FollowOption/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react"
+import { navigate } from "gatsby"
+import FollowOption from "./component/Users"
+
+import { BsArrowLeft } from "react-icons/bs"
+import { users } from "../../data/data"
+import {
+  StyledFollowOptionHeader,
+  StyledFollowOptionInput,
+  StyledFollowOptionPageCt,
+  StyledFollowOptionSearchBox,
+  StyledFollowOptionPopUp,
+  StyledFollowOptionTogglePagesBar,
+  StyledUsersListCt,
```

Comment:
What is `Ct`? it is better to write the full names, and if you think that `Styled` prefix is too long and is annoying, you can use `St` prefix instead, like `StUserWrapper`.